### PR TITLE
1120 read only file system

### PR DESCRIPTION
**Pull request by nilsgstrabo**

### CODE REVIEW

File: pkg/apis/radixvalidators/validate_ra.go [hunk 1]
```diff
@@ -307,6 +307,11 @@ func validateComponents(app *radixv1.RadixApplication) error {
 			errs = append(errs, err)
 		}
 
+		/*err = validateUseReadOnlyFileSystem(component.UseReadOnlyFileSystem)
```
Comment (anneliawa) [inline pkg/apis/radixvalidators/validate_ra.go:310]: remove?

**Review by Richard87 — CHANGES_REQUESTED** [review]
Added some test coments

**Review by Richard87 — DISMISSED** [review]
Godkjent

**Review by Richard87 — APPROVED** [review]
Godkjent! :D

**Pull request merged**